doc/installation.rst

```diff
@@ -75,13 +75,14 @@ On macOS, open the disk image and drag Spyder to your :guilabel:`Applications` f
 
 .. table::
 
-   ================ ================ ================
-   `Windows`_       `macOS M1`_      `macOS Intel`_
-   ================ ================ ================
-
-.. _Windows: https://github.com/spyder-ide/spyder/releases/latest/download/Spyder_64bit_full.exe
-.. _macOS M1: https://github.com/spyder-ide/spyder/releases/latest/download/Spyder_arm64.dmg
-.. _macOS Intel: https://github.com/spyder-ide/spyder/releases/latest/download/Spyder.dmg
+   ================ ================ ================ ================
+   `Windows`_       `Linux`_         `macOS M1`_      `macOS Intel`_
+   ================ ================ ================ ================
+
+.. _Windows: https://github.com/spyder-ide/spyder/releases/latest/download/Spyder-Windows-x86_64.exe
+.. _Linux: https://github.com/spyder-ide/spyder/releases/latest/download/Spyder-Linux-x86_64.sh
```

Comment:
I think we should move the link to Linux to the end of the row because it's our lease popular OS.

Reply:
Well the main reason I put Linux before Mac was that because there were two Mac installers, so it would be more buried there.

And as far as I have been able to tell Linux is not our least popular OS—my informal observations triaging user issues suggested that Linux was as or more popular than macOS for us (Windows was most popular, of course), and a quick issue search supports that— [2.4k hits for `mac OR macos`](https://github.com/search?q=repo%3Aspyder-ide%2Fspyder+mac+OR+macos&type=issues) vs [3.6k for linux OR ubuntu](https://github.com/search?q=repo%3Aspyder-ide%2Fspyder+linux+OR+ubuntu&type=issues).

In terms of [Conda-Forge downloads](https://anaconda.org/conda-forge/spyder/files), looking at the latest release that's been out a while, 5.5.5, its the same story: 13 103 for Linux x64 (+ 684 Linxu ARM) vs 5220 for macOS ARM and 2682 for macOS x64 (total 8082). Linux x64 has 60% higher numbers than M1 and nearly 5x that of x64. And same thing on PyPI if not more so; Linux regularly has 5x+ the daily downloads of macOS there.

Yup, I spent nearly 30 minutes pulling and double-checking all the data on this, but that's because its quite useful to have an accurate, data-driven picture of what our most popular OSes are, far beyond just this particular minor discussion.

Reply:
Changed it anyway, on your insistence.